lib/content-services/src/lib/search/services/search.service.ts

```diff
@@ -51,12 +48,14 @@ export class SearchService {
      * @param options Options for delivery of the search results
      * @returns List of nodes resulting from the search
      */
-    getNodeQueryResults(term: string, options?: SearchOptions): Observable<NodePaging> {
+    getNodeQueryResults(term: string, options?: FindNodesQuery): Observable<NodePaging> {
```

Comment:
could you please correct all references to SearchOptions type in search.service.md file? Change them to FindNodesQuery